script.js

```
@@ -1,84 +1,362 @@
+            }, 2500);
+        }
+    }
+    // --- FIN LÓGICA DE ALERTAS TEMPORALES ---
+
+    // --- INICIO LÓGICA DE TRADUCCIÓN ---
+    function generateMenuItemKey(originalItemName, type) {
+        const baseKey = originalItemName.toLowerCase()
+            .normalize("NFD").replace(/[\u0300-\u036f]/g, "")
+            .replace(/ñ/g, 'n')
+            .replace(/\s+/g, '_')
+            .replace(/[^a-z0-9_]/g, '');
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)